libindy_vdr/src/resolver/did_document.rs

```diff
@@ -173,12 +174,52 @@ fn merge_diddoc(base: &mut SJsonValue, content: &SJsonValue) {
     }
 }
 
-fn expand_verkey(id: &str, verkey: &str) -> String {
-    if let Some(stripped) = verkey.strip_prefix('~') {
-        format!("{}{}", id, stripped)
+// Returns raw verkey in case of conversion errors, otherwise 'default' indy handling
+pub fn expand_verkey(id: &str, verkey: &str) -> String {
+    let expanded_verkey = expand_verkey_indy(id, verkey);
+    if expanded_verkey.is_err() {
+        return verkey.to_string();
+    }
+    return expanded_verkey.unwrap();
```

Comment:
I think a match would be cleaner here:

```suggestion
    match expand_verkey_indy(id, verkey) {
      Ok(key) => key,
      Err(_) => verkey.to_string()
    }
```

returning with a `&str` would make it even terser though.

Reply:
Thanks! I fully agree - looking at your code feels like looking at rust code - mine feels like Go :D

Reply:
```rust
expand_verkey_indy(id, verkey).unwrap_or(verkey.to_string)
```

Would probably work as well now that I think about it.